### PR TITLE
Serialize block data when sending over the network - Closes #5007

### LINKED ISSUE #5007: Serialize block and transaction data when sending over the network
### Expected behavior

A node should be able to send serialized transactions and blocks when requested from the network

### Actual behavior

Its not sending serialized block and transaction data when requested by a peer from the network

```javascript
15:42:14 ERROR lisk-framework: Peer request not fulfilled event: Could not respond to peer request getHighestCommonBlock because of error: Do not know how to serialize a BigInt (module=lisk:app)
TypeError: Do not know how to serialize a BigInt
    at JSON.stringify (<anonymous>)
    at Object.module.exports.encode (/Users/ishan/Documents/lisk/lisk-sdk/node_modules/sc-formatter/index.js:93:15)
    at SCTransport.encode (/Users/ishan/Documents/lisk/lisk-sdk/node_modules/socketcluster-client/lib/sctransport.js:384:21)
    at Response._respond (/Users/ishan/Documents/lisk/lisk-sdk/node_modules/socketcluster-client/lib/response.js:15:34)
    at Response.end (/Users/ishan/Documents/lisk/lisk-sdk/node_modules/socketcluster-client/lib/response.js:27:10)
    at Response.callback (/Users/ishan/Documents/lisk/lisk-sdk/node_modules/socketcluster-client/lib/response.js:51:10)
    at /Users/ishan/Documents/lisk/lisk-sdk/node_modules/socketcluster-client/lib/scclientsocket.js:643:27
    at P2PRequest._respondCallback (/Users/ishan/Documents/lisk/lisk-sdk/elements/lisk-p2p/dist-node/p2p_request.js:23:13)
    at P2PRequest.end (/Users/ishan/Documents/lisk/lisk-sdk/elements/lisk-p2p/dist-node/p2p_request.js:48:14)
    at P2P.<anonymous> (/Users/ishan/Documents/lisk/lisk-sdk/framework/src/application/network/network.js:256:13)
15:42:14 FATAL lisk-framework: System error: unhandledRejection (module=lisk:app)
Message: A response has already been sent for the request procedure <<getHighestCommonBlock>> 
 Trace: ResponseAlreadySentError: A response has already been sent for the request procedure <<getHighestCommonBlock>>
```

### Steps to reproduce

Run the application and connect to some nodes

### Which version(s) does this affect? (Environment, OS, etc...)
 
`4.0.0-alpha.0`

### PR DESCRIPTION
**Pull request by ishantiw**
### What was the problem?

This PR resolves #5007 

### How was it solved?

Serialize the data before sending to the network 
These actions in `node` were responding with deserialized data and the application was failing.

`getHighestCommonBlock`
`getHighestCommonBlock`

### How was it tested?

`node test/test_app` and connect with multiple nodes

### CODE REVIEW

File: framework/src/application/node/node.js [hunk 1]
```diff
@@ -318,14 +318,18 @@ module.exports = class Node {
 			}),
 			getLastBlock: async () => this.processor.serialize(this.chain.lastBlock),
 			getBlocksFromId: async action =>
-				this.transport.handleRPCGetBlocksFromId(
-					action.params.data,
-					action.params.peerId,
+				this.processor.serialize(
+					this.transport.handleRPCGetBlocksFromId(
```
Comment (ManuGowda) [inline framework/src/application/node/node.js]: inside this function we have called serialise `return blocks && blocks.map(block => this.chainModule.serialize(block));`
so i dont know if this is correct

File: framework/src/application/node/node.js [hunk 2]
```diff
@@ -318,14 +318,18 @@ module.exports = class Node {
...
+						action.params.data,
+						action.params.peerId,
+					),
 				),
 			getHighestCommonBlock: async action =>
-				this.transport.handleRPCGetGetHighestCommonBlock(
-					action.params.data,
-					action.params.peerId,
+				this.processor.serialize(
+					this.transport.handleRPCGetGetHighestCommonBlock(
```
Comment (ManuGowda) [inline framework/src/application/node/node.js]: for this one it dint have.. so maybe just follow the same pattern add the `serialize` in `handleRPCGetGetHighestCommonBlock` function? framework/src/application/node/transport/transport.js

File: framework/src/application/node/node.js (hunk 1 above)
Comment (ishantiw) [reply framework/src/application/node/node.js]: Oh yeah this is not relevant, I thought it followed the same pattern as `handleRPCGetGetHighestCommonBlock`

File: framework/src/application/node/transport/transport.js [hunk 3]
```diff
@@ -155,7 +155,7 @@ class Transport {
 
 		const commonBlock = await this.chainModule.getHighestCommonBlock(data.ids);
 
-		return commonBlock;
+		return this.processorModule.serialize(commonBlock);
```
Comment (ManuGowda) [inline framework/src/application/node/transport/transport.js]: this.chainModule.serialize

**Pull request merged**